### PR TITLE
Add example activity of creating MapView

### PR DESCRIPTION
**Pull request by kiryldz**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: #83 
`<changelog>Add example activity of creating MapView</changelog>`

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [x] Include before/after visuals or gifs if this PR includes visual changes.
 - [x] Write tests for all new functionality. If tests were not written, please explain why.
 - [x] Add example if relevant.
 - [x] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [x] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### CODE REVIEW

File: app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_map_view_customization)
+    configureMapViewFromXml()
+    configureMapViewFromCode()
+  }
+
+  private fun configureMapViewFromCode() {
+    val mapboxMapOptions = MapboxMapOptions(this)
```
Comment (Chaoba) [inline app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt:26]: nit: I would prefer to use apply block here.

File: app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  private fun configureMapViewFromXml() {
+    // let's set `custom` token to MapView from code (however it will be same token from resources so that map will work)
+    MapboxOptions.setDefaultResourceOptions(this, getString(R.string.mapbox_access_token))
+    // all options provided in xml file - so we just load style
+    mapView.getMapboxMap().loadStyleUri(Style.DARK)
+  }
+
+  override fun onStart() {
+    super.onStart()
+    mapView.onStart()
```
Comment (Chaoba) [inline app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt:67]: customMapView should also be handled in the life cycle methods

Comment (kiryldz) [reply app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt:67]: you're totally correct!
thanks!

File: app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt (hunk 1 above)
Comment (kiryldz) [reply app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt:26]: 👍

File: app/src/main/res/layout/activity_map_view_customization.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        app:mapbox_cameraTargetLat="52.52"
+        app:mapbox_cameraTargetLng="13.4050"
+        app:mapbox_cameraPitch="65"
+        app:mapbox_cameraZoom="15"
+        app:mapbox_mapOrientation="downwards"
+        app:mapbox_mapSurface="surface_view"
+        tools:context=".examples.SimpleMapActivity" />
```
[the remark this answers is not in the document]
Comment (kiryldz) [reply app/src/main/res/layout/activity_map_view_customization.xml]: well, it's pretty simple actually 😄 
fixed

**Review by Chaoba — APPROVED** [review]
LGTM

**Review by pengdev — APPROVED** [review]

**Pull request merged**